canonical/canonical.com
# Fix blog department pages

## Done

Fix blog links as mentioned in https://github.com/canonical/canonical.com/issues/972
Fix blog article template setting `body_class` twice
Add paper background body class to all department pages.

## QA

- Check blog links on department pages work now  https://canonical-com-980.demos.haus/careers/engineering, https://canonical-com-980.demos.haus/careers/web-and-design

If the blog gives you a 500, it's because it's going to canonical.com, the fix is in this PR, so to check that you'd need to change the URL to `https://canonical-com-980.demos.haus/blog/<whatever-blog-you-clicked>`


## Issue / Card

Fixes https://github.com/canonical/canonical.com/issues/972
